content/techniques/caching.md

```diff
@@ -18,6 +18,7 @@ Nest provides a unified API for various cache storage providers. The built-in on
 In order to enable caching, import the `CacheModule` and call its `register()` method.
 
 ```typescript
+@@filename(app.module)
```

Comment:
What benefits does this change bring given the fact that class names are shown in the example (so it's pretty clear from the context what the file is)?

Reply:
For beginners it's perhaps not always clear and it would be more consistent with other examples. Having always a file name wouldn't really bloat the docs.

Reply:
This should be clear at that point for everyone (if someone is looking at the Cache section already). IMO having always a file name would bloat the docs without any extra benefit